frontend/packages/pipelines-plugin/integration-tests/support/pages/pipelines/task-page.ts

```diff
@@ -0,0 +1,60 @@
+import * as yamlEditor from '@console/cypress-integration-tests/views/yaml-editor';
+import { pipelineBuilderPO } from '../../page-objects/pipelines-po';
+
+export const tasksPage = {
+  clickOnCreateTask: () => {
+    cy.get('[data-test-id="dropdown-button"]').click();
+    cy.get('[data-test-dropdown-menu="tasks"]').click();
+    cy.get("[class='mtk26']").should('exist');
```

Comment:
Can this be done using less obscure locator? Or using something like `app.waitForLoad`?

Reply:
class='mtk26' captures the whole YAML editor. There was one instance where the page loaded and the editor didn't. So I tried this way

Reply:
`mtk26` is a paragraph span, that's like 6 levels below the `class="yaml-editor"` that actually wraps the whole editor

My point being it wasn't really clear what the element represents until I specifically checked the DOM.

Reply:
Agreed @jrichter1. Shall I make the changes